lib/gcpspanner/feature_search_query.go

```diff
@@ -303,6 +305,31 @@ func (b *FeatureSearchFilterBuilder) featureNameFilter(featureName string, op se
 		opStr, paramName)
 }
 
+func (b *FeatureSearchFilterBuilder) featureDescriptionFilter(
+	description string, op searchtypes.SearchOperator) string {
+	// Normalize the string to lower case to use the computed column.
+	description = strings.ToLower(description)
+	// Safely add the database % wildcards if they do not already exist.
+	if !strings.HasPrefix(description, "%") {
```

Comment:
Are wildcards available for all ops?

Reply:
Good question. Actually I don't think so.  Also, currently, we take the "=" and convert to the `LIKE` operator. I will actually add a new [operator](https://github.com/GoogleChrome/webstatus.dev/blob/e0400422e0fb365e85165e4fa793fa57e256df7e/lib/gcpspanner/searchtypes/searchtypes.go#L42-L50) that this filter and the existing feature name [filter](https://github.com/GoogleChrome/webstatus.dev/blob/main/lib/gcpspanner/feature_search_query.go#L287) can use since they both could benefit from that. With this new operator, I can explicitly know if I should add the wildcard to the strings.

Reply:
I'll do this in a follow up PR shortly

Reply:
Created https://github.com/GoogleChrome/webstatus.dev/pull/1593